run_performance_tests.m

```diff
@@ -0,0 +1,36 @@
+function run_performance_tests(commit)
+    disp(version);
+    if ~exist('spinw', 'class')
+        if exist('swfiles', 'dir') && exist('external', 'dir') && exist('dat_files', 'dir')
+            addpath(genpath('swfiles'));
+            addpath(genpath('external'));
+            addpath(genpath('dat_files'));
+        else
+            error(['SpinW is not installed and the swfiles, external and/or ', ...
+                   'dat_files drectories couldn''t be found on the current ', ...
+                   'path, so the tests cannot be run.'])
+        end
+    end
```

Comment:
Should this bit be moved below, to just before the `% run tests` part? If we do this check now, if something goes wrong with the following commit stuff these paths may no longer be valid